docs/standalone/building-on-electrumsv/node-wallet-api.rst

```diff
@@ -749,16 +749,12 @@ call processing are described above.
 
     - :Code: -32700 ``RPC_PARSE_ERROR``
       :Message: | ``JSON value is not a null as expected``
-                | This is an intentional incompatibility as we do not support this parameter. The
-                  type of the entries in the ``account`` parameter are expected to be strings and
-                  one or more were interpreted as another type.
+                | This is an intentional incompatibility as we do not support this parameter.
```

Comment:
Because isn't it weird to have in our documentation about the 'account' parameter being expected to be strings etc. but the code will error if you provide a string - it isn't supported and needs to be null.

Reply:
Yes.